Currently Loaded Modulefiles:
  1) python/3.7
Traceback (most recent call last):
  File "deterministic-super-tight-novmax.py", line 64, in <module>
    experiment()
  File "deterministic-super-tight-novmax.py", line 59, in experiment
    parallel_run=True, n_processes=None)
  File "/home/disc/e.lecarpentier/git/llrl/llrl/experiments.py", line 110, in run_agents_lifelong
    lifelong_plot(agents, path, n_tasks, n_episodes, confidence, open_plot, plot_title)
  File "/home/disc/e.lecarpentier/git/llrl/llrl/utils/chart_utils.py", line 25, in lifelong_plot
    dfs.append(pandas.read_csv(agent_path))
  File "/home/disc/e.lecarpentier/.conda/envs/myenv/lib/python3.7/site-packages/pandas/io/parsers.py", line 685, in parser_f
    return _read(filepath_or_buffer, kwds)
  File "/home/disc/e.lecarpentier/.conda/envs/myenv/lib/python3.7/site-packages/pandas/io/parsers.py", line 463, in _read
    data = parser.read(nrows)
  File "/home/disc/e.lecarpentier/.conda/envs/myenv/lib/python3.7/site-packages/pandas/io/parsers.py", line 1154, in read
    ret = self._engine.read(nrows)
  File "/home/disc/e.lecarpentier/.conda/envs/myenv/lib/python3.7/site-packages/pandas/io/parsers.py", line 2048, in read
    data = self._reader.read(nrows)
  File "pandas/_libs/parsers.pyx", line 879, in pandas._libs.parsers.TextReader.read
  File "pandas/_libs/parsers.pyx", line 894, in pandas._libs.parsers.TextReader._read_low_memory
  File "pandas/_libs/parsers.pyx", line 948, in pandas._libs.parsers.TextReader._read_rows
  File "pandas/_libs/parsers.pyx", line 935, in pandas._libs.parsers.TextReader._tokenize_rows
  File "pandas/_libs/parsers.pyx", line 2130, in pandas._libs.parsers.raise_parser_error
pandas.errors.ParserError: Error tokenizing data. C error: Expected 5 fields in line 107, saw 9

